### PR TITLE
Recipe for nuboxdevcom/maintenance-bundle

### PR DESCRIPTION
**Pull request by NuBOXDevCom**
| Q             | A
| ------------- | ---
| License       | MIT

Recipe for nuboxdevcom/maintenance-bundle@2.*

### CODE REVIEW

**Comment by NuBOXDevCom** [comment]
Is it possible to merge my PR to master?

File: nuboxdevcom/maintenance-bundle/2.0/manifest.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "NDC\\MaintenanceBundle\\NDCMaintenanceBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "composer-scripts": {
```
Comment (Nyholm) [inline nuboxdevcom/maintenance-bundle/2.0/manifest.json]: You do not need to include the composer-scripts. They are already included by other bundles (framework bundle)

**Comment by NuBOXDevCom** [comment]
Thank you for the feedback. manifest.json as corrected ;)

**Comment by Nyholm** [comment]
Great

**Pull request merged**